backend/endpoints/messages.py

```diff
@@ -204,3 +205,32 @@ async def update_message(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail={"message not edited": edited_message},
     )
+
+
+@router.get(
+    "/org/{org_id}/rooms/{room_id}/messages",
+    response_model=ResponseModel,
+    status_code=status.HTTP_200_OK,
+    responses={424: {"detail": "ZC Core failed"}},
+)
+async def get_messages(org_id, room_id):
+    """Fetches all messages sent in a particular room.
+
+    Args:
+        org_id (str): A unique identifier of an organization
+        room_id (str): A unique identifier of the room where messages are fetched from
+
+    Returns:
+        A list of message objects
+
+    """
+    response = await get_room_messages(org_id, room_id)
+    if response is None or "status_code" in response:
+        raise HTTPException(
+            status_code=status.HTTP_424_FAILED_DEPENDENCY,
+            detail=response,
```

Comment:
I think the ZC core failed message should be added to the returned response.

Reply:
Ok. We'll handle the custom exception later. @pauline-banye